Proposal for the basic structure of a simulation study.md

```diff
@@ -0,0 +1,49 @@
+# Proposal for the basic structure of a simulation study
+
+### Final Structure
+
+1. *Defining aims & objectives (including hypotheses)*
+2. *Simulation procedure /Design & Data-generation mechanism*
+3. *Methods selection (including software and estimands)*
```

Comment:
I agree with this step to explicitly include estimands selection in a separate step (apart from experimental design, population model specification and population values)
I would suggest to treat software selection as a separate step.